packages/test-app/tests/integration/components/truncate-text-test.js

```diff
@@ -8,138 +8,107 @@ module('Integration | Component | truncate-text', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders empty', async function (assert) {
-    await render(hbs`<TruncateText />
-`);
+    await render(hbs`<TruncateText />`);
     assert.strictEqual(component.text, '');
 
-    await render(hbs`<TruncateText></TruncateText>
-`);
+    await render(hbs`<TruncateText />`);
     assert.dom(this.element).hasText('');
   });
 
   test('it renders short text in full', async function (assert) {
     const shortText = 'template block text';
     this.set('shortText', shortText);
 
-    await render(hbs`<TruncateText @text={{this.shortText}} />
-`);
+    await render(hbs`<TruncateText @text={{this.shortText}} />`);
     assert.strictEqual(component.text, shortText);
     assert.notOk(component.collapse.isVisible);
     assert.notOk(component.expand.isVisible);
 
-    await render(hbs`
-      <TruncateText>
-        {{this.shortText}}
-      </TruncateText>
-
-`);
+    await render(hbs`<TruncateText>
+  {{this.shortText}}
+</TruncateText>`);
```

Comment:
I guess this is an improvement, but it still seems awkward.

Reply:
One of the best/worst things about prettier is that it makes a bunch of choices for you. Sometimes they're awesome, sometimes you hate them. I try and lean into the good part - we don't have to make any of these decisions ourselves.